HeterogeneousCore/SonicTriton/interface/TritonData.h

```diff
@@ -40,6 +42,13 @@ class TritonData {
   bool setShape(const ShapeType& newShape) { return setShape(newShape, true); }
   bool setShape(unsigned loc, int64_t val) { return setShape(loc, val, true); }
 
+  void setConverterParams(const edm::ParameterSet& conf) {
+    converterConf_ = conf;
+    converterName_ = conf.getParameter<std::string>("converterName");
```

Comment:
my original thought was that the bit specifications for ap_fixed could be given in the pset, but having now tried it there are some real difficulties with doing that. Ill remove it now and if somewhere down the line we need to add it back its pretty easy

Reply:
Actually I spoke too soo, it looks like the factory `create()` function expects 2 args, so we are perhaps better off just keeping the basically empty pset around

Reply:
The signature of that function is `create(const std::string& iName, Args... args)`. The first argument is used for lookup, while the second is variadic, and just has to match a constructor for the class to be built. For now, you could just pass the name rather than keeping the full PSet around.

Reply:
Ok, thanks for that explanation, I didnt fully appreciate where `args` was being used. I have this working now